### PR TITLE
WIP: added some findBySlug tests

### PR DESCRIPTION
**Pull request by gabrielperales**
Hi, I've seen that you are looking for add some tests to your code, and I would like to help out closing some issues! so I have started doing some tests but I have the feeling that everything is too verbose. Do you think I should follow another approach?

### CODE REVIEW

File: tests/ContentUtilsTests.elm [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                        content =
+                            []
+
+                        expected =
+                            Nothing
+
+                        actual =
+                            findBySlug content "whatever"
+                    in
+                        Expect.equal expected actual
```
Comment (jackfranklin) [inline tests/ContentUtilsTests.elm]: I think I'd be inclined to inline more of this

```elm
Expect.equal Nothing <| findBySlug content "whatever"
```

Would work I think?

**Comment by jackfranklin** [comment]
Hey, left a comment - I think you can use `<|` and a bit less `let...in` (especially when the values are so small) to clean it up. What do you think?

Thanks :)

**Comment by gabrielperales** [comment]
Cool, I think you are right. I'm going to change that and keep adding some tests :)

**Comment by gabrielperales** [comment]
I've been trying to test other functions, but for example:

```elm
postsInOrder : List Content
postsInOrder =
    List.sortWith (flipComparison contentByDateComparison) Posts.posts
```

`postInOrder` depends on `Posts.posts` and those posts are going to be changing each time you want to write a new post, and that could break our tests. In my opinion `postInOrder` should receive those posts as a parameter:

```elm
postsInOrder : List Content -> List Content
postsInOrder =
    List.sortWith (flipComparison contentByDateComparison)
```

The same thing with `watchMeElmPosts`. What do you think?

**Comment by jackfranklin** [comment]
Yeah I would agree that they should take the posts. The current way isn't very Elm like.

**Pull request merged**